src/views/Profile.vue

```diff
@@ -132,43 +190,120 @@ a {
   max-height: 150px;
 }
 
+/*
+  options nav
+*/
 .options-nav {
-  margin-top: 2rem;
+  margin-top: 1.5rem;
   padding: 0;
   display: flex;
+  flex-direction: column;
   justify-content: space-around;
   position: relative;
   list-style: none;
 
   &::before {
     content: '';
     position: absolute;
-    bottom: 0;
+    top: 0;
     left: 0;
     width: 100%;
     height: 1px;
-    background-color: $black;
+    background-color: $gray-400;
   }
 
-  .option-name[aria-selected='true'] .option-text {
-    color: $primary;
-    border-bottom: 3px solid $primary;
+  a {
+    cursor: pointer;
+    display: flex;
+    height: 100%;
+    width: 100%;
   }
 
-  .option-name .option-text {
-    cursor: pointer;
-    display: inline-block;
-    font-size: 1rem;
-    color: #b9b9b9;
-    padding-bottom: 1rem;
-    margin-right: 0;
-    letter-spacing: 0.25rem;
-    text-indent: 0.25rem;
+  .option-name {
+    display: flex;
+    height: 4rem;
+
+    &[aria-selected='true'] {
+      border-left: 3px solid $blue;
+      background-color: #f6f6f8;
+    }
+
+    .option-text {
+      display: inline-block;
+      font-size: 1.125rem;
+      letter-spacing: 0.25rem;
+      text-indent: 0.25rem;
+      vertical-align: middle;
+      padding-left: 4rem;
+      margin: auto 0;
+    }
   }
 }
 
+.edited-article {
+  background: left 2rem top 50% url('../assets/icons/ic-edited.svg') no-repeat;
+  padding-left: 4.5rem;
+}
+
+.history {
+  background: left 2rem top 50% url('../assets/icons/ic-history.svg') no-repeat;
+  padding-left: 4.5rem;
+}
+
+.collection {
+  background: left 2rem top 50% url('../assets/icons/ic-bookmark.svg') no-repeat;
+  padding-left: 4.5rem;
+}
+
+.setting {
+  background: left 2rem top 50% url('../assets/icons/ic-settings.svg') no-repeat;
+  padding-left: 4.5rem;
+}
+
 .profile-title {
   margin: 1rem 0;
-  color: rgb(49, 87, 211)
+  color: rgb(49, 87, 211);
+}
+
+.personal-status {
+  table-layout: fixed;
+  margin-left: 2rem;
+  width: calc(300px - 2rem);
+
+  &-first-col {
```

Comment:
Maybe we can use [:first-child](https://developer.mozilla.org/zh-TW/docs/Web/CSS/:first-child#compat-mobile), instead of a class, which is compatible on common devices.

Reply:
Done 😎